crates/apollo-encoder/src/enum_value.rs

```diff
@@ -1,6 +1,6 @@
 use std::fmt;
 
-use crate::StringValue;
+use crate::{Directive, StringValue};
```

Comment:
nit: I saw you grouped Field Definition and Field in the same file, we can probably do the same for Enum Definition and Enum Value.

Reply:
We also have `field.rs` and `field_value.rs` I think it's good to keep the same. If you think it's weird then I can also delete the `field_value.rs` and put everything inside. As you wish

Reply:
ah! i missed the field.rs and field_value.rs differentiation. let's keep it separate then